samples/profiler/sample.c

```diff
@@ -47,10 +50,10 @@ mono_profiler_init_sample (const char *desc)
 {
 	MonoProfiler *prof;
 
-	prof = g_new0 (MonoProfiler, 1);
+	prof = calloc(1, sizeof(MonoProfiler));
```

Comment:
While you're here, could you make this a static variable instead of allocating it on the heap?